plugins/esxi_logs.yaml

```diff
@@ -0,0 +1,109 @@
+version: 0.0.1
+title: VMware ESXi
+description: Log parser for VMware ESXi
+parameters:
+  - name: listen_port
+    label: Listen Port
+    description: A port which the agent will listen for syslog messages
+    type: int
+    default: 5140
+  - name: listen_ip
+    label: Listen IP
+    description: A syslog ip address of the form `<ip>`
+    type: string
+    default: "0.0.0.0"
+  - name: enable_tls
+    type: bool
+    default: false
+  - name: certificate_file
+    type: string
+    default: "/opt/cert"
+  - name: private_key_file
+    type: string
+    default: "/opt/key"
+
+template: |
+  receivers: 
+    tcplog:
+      listen_address: '{{ .listen_ip }}:{{ .listen_port }}'
+      attributes:
+        log_type: vmware_esxi
+      add_attributes: true
+      {{ if .enable_tls }}
+      tls:
+        certificate: {{ .certificate_file }}
+        private_key: {{ .private_key_file }}
```

Comment:
These parameters should not be certificate and private_key, but cert_file and key_file: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/tcplogreceiver#tls-configuration